client/package.json

```diff
@@ -18,7 +18,8 @@
     "react-moment": "^1.1.1",
     "react-router-dom": "^5.3.0",
     "react-scripts": "4.0.3",
-    "web-vitals": "^1.1.2"
+    "web-vitals": "^1.1.2",
+    "react-qrcode-logo": "^2.9.0"
```

Comment:
what's this

Reply:
Herman added this in #41, but to the server's package.json. Seems to have worked but not now when the Dockerfile builds the frontend and backend in separate containers. I just moved it to the client's package.json